### DISCUSSION TITLE
Replacing bot commands with slash commands and adding ephemeral messages

### DISCUSSION THREAD

**Discussion by SatireSage** [CSSS/wall_e#386] [Ideas]
I believe the following features would be a great addition to the bot:

1. Replacing a Discord bot with slash commands involves updating the bot's existing command structure to a new system using slashes. This new system is designed to work more smoothly with Discord's interface, allowing users to easily access commands and receive immediate feedback through ephemeral messages.

2. Ephemeral messages are private messages sent to the user and only visible to them for a limited time. This feature is particularly useful for sensitive or private information as it is not seen by others, including administrators. 

By incorporating ephemeral messages, users can receive real-time feedback on their actions, reducing wait times and improving the overall user experience. Additionally, with slash commands, users don't have to memorize complex command sequences; instead, the commands and their applicable options are presented to them in an intuitive user interface. 

**Images for reference:**
<img width="1155" alt="Screenshot 2023-03-26 at 6 59 15 PM" src="https://user-images.githubusercontent.com/49692422/227822954-6366f884-c91a-4b09-a992-b3ddb1a559f7.png">
<img width="350" alt="Screenshot 2023-03-26 at 6 59 41 PM" src="https://user-images.githubusercontent.com/49692422/227823003-f5e0816f-bb97-4ca1-bdaa-b24a15646dfa.png">
<img width="1167" alt="Screenshot 2023-03-26 at 6 59 48 PM" src="https://user-images.githubusercontent.com/49692422/227823012-c58c77ab-b249-4f3b-b289-1acab0d8b4d6.png">

**Comment by modernNeo** [comment]
You are free to implement these changes yourself and I encourage you to 😄 .

**Reply by SatireSage** [reply]
> You are free to implement these changes yourself and I encourage you to 😄 .

Awesome! Thank you so much!!
